addons/pos_self_order/static/src/app/services/self_order_service.js

```diff
@@ -885,10 +885,10 @@ export class SelfOrder extends Reactive {
         return this.getAvailableCategories().length > 1;
     }
 
-    get kioskBackgroundImage() {
-        const bgImage = this.config._self_ordering_image_background_ids[0];
-        if (bgImage) {
-            return `url(data:image/png;base64,${bgImage.data})`;
+    get kioskBackgroundImageUrl() {
+        const imageId = this.config._self_ordering_image_background_ids[0];
+        if (imageId) {
+            return `url('/web/image/ir.attachment/${imageId}/raw')`;
```

Comment:
Why its a getter here ? We cannot use `this.config._self_ordering_image_background_ids[0]` directly ?

You don't have to change that, it's just a comment.